Google Drive Config for WIP (#4493)
MicrosoftDocs/windows-itpro-docs: windows/security/information-protection/windows-information-protection/testing-scenarios-for-wip.md

            </ul>
        </td>
    </tr>
    <tr>

>         <td>Stop Google Drive to sync WIP protected files and folders.</td>

Reviewer: ```suggestion
        <td>Stop Google Drive from syncing WIP protected files and folders.</td>
```

        <td>
            <ul>

>                 <li>In silent configuration, add Google Drive in Protected Apps and set it to Deny. This way, Google Drive will not sync WIP protected files and folders.</li>

Reviewer 2: ```suggestion
                <li>In silent configuration, add Google Drive to Protected Apps and set it to Deny. This way, Google Drive will not sync WIP protected files and folders.</li>
```

Reviewer 3: @joinimran please check. Thanks.

>                 <li>In silent configuration add Google Drive in Protected Apps and set it to Deny. This way Google Drive will not sync                       WIP protected files and folders.</li>

Reviewer: ```suggestion
                <li>In silent configuration add Google Drive in Protected Apps and set it to Deny. This way Google Drive will not sync WIP protected files and folders.</li>
```
